src/common/types.hpp

```diff
@@ -79,8 +81,19 @@ enum otbrError
     OTBR_ERROR_PARSE           = -8,  ///< Parse error.
     OTBR_ERROR_NOT_IMPLEMENTED = -9,  ///< Not implemented error.
     OTBR_ERROR_INVALID_ARGS    = -10, ///< Invalid arguments error.
+    OTBR_ERROR_DUPLICATED      = -11, ///< Duplicated operation, resource or name.
 };
 
+/**
+ * This function converts `otbrError` to `otError`.
+ *
+ * @param[in]  aError  A OTBR error.
+ *
+ * @returns  The corresponding OT error.
+ *
+ */
+otError OtbrErrorToOtError(otbrError aError);
```

Comment:
This is because the `otbrError` enum itself is out of the `otbr` namespace, so I am keeping the conversion method with it. I think `otbrError` is out of the namespace maybe because `otbr::otbrError` seems too verbose?